### ISSUE TITLE
Rancher CLI --prune does not display the services being deleted in the command line

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#10110]
**Rancher versions:**
rancher/server:v1.6.11-rc3
Rancher CLI: v0.6.5-rc1

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**Ubuntu

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**Digital OCean

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single noce

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**Cattle

**Steps to Reproduce:**
1. Create a stack "teststack" through CLI  using the compose file
```
version: '2'
services:
  service1:
    image: soumyalj/nginx
    stdin_open: true
    tty: true
  service2:
    image: soumyalj/nginx
    stdin_open: true
    tty: true
    labels:
      io.rancher.container.pull_image: always
  service3:
    image: soumyalj/nginx
    stdin_open: true
    tty: true

```
Use --prune through CLI with an updated compose file as below:
teststack1.yml
```
version: '2'
services:
  service1:
    image: soumyalj/nginx
    stdin_open: true
    tty: true
```

```
rancher-v0.6.5-rc1 soumya$ ./rancher up -d --stack teststack --prune --file teststack1.yml 
INFO[0002] [service1]: Creating                         
INFO[0004] [service1]: Created                          
INFO[0004] [service1]: Starting                         
1s172
INFO[0004] [service1]: Started             

```
The CLI does not display that the services service2 and service3 that are being pruned(deleted). There is no indication to the user about the services being pruned.

**Comment by deniseschannon** [comment]
Available with v0.6.5-rc2

**Comment by sangeethah** [comment]
Tested with rancher cli version - v0.6.5-rc3 .
Information about services that get deleted when using up command with  ```--prune``` option is presented to the user.

```INFO[0001] Deleting service lb-2                        
INFO[0001] Deleting service testtarget2                 
INFO[0001] Deleting service testtarget4                 
INFO[0001] Deleting service testaf ```
